customs_invoice_report/README.rst

```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Custom Invoice Report
+=====================
+         
+Add 'Customs Invoice Report' to standard report
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `<https://github.com/Elico-Corp/openerp-7.0/pull/90>`_. 
+In case of trouble, please check there if you issue has been already reported.
+if you spotted it first,help us smash it by providing detailed and welcomed 
+feedback.
+
+Contributors
+------------
+
+Eric Caudal <eric.caudal@elico-corp.com>
```

Comment:
use "* " to start the list:
`* Eric Caudal <eric.caudal@elico-corp.com>`
`* Jon Chow <jon.chow@elico-corp.com>`